### ISSUE TITLE
Alveolar capillary dysplasia

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#6228]
**Preferred term label:**
Alveolar capillary dysplasia

**Synonyms**

**Definition (free text, please give PubMed ID)**
A developmental defect of the alveolar capillaries characterized by a decreased number of pulmonary capillaries located away from the alveolar epithelium, thickened alveolar septae, medial hypertrophy of small pulmonary arteries and muscularization of distal arterioles, and in some cases lymphangiectasis. PMID: 21471096

**Parent term (use hpo.jax.org/app)**
Do we need a term for alveolar capillaries underneath https://hpo.jax.org/app/browse/term/HP:0004930  ?

@crapp, @mgriese, @katrinknoflach

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

PMID:19500772 7/14

**Your nano-attribution (ORCID)**

**Comment by mgriese** [comment]
This is extremely difficult; first I thought that you should not create this term, as not commonly used/defined. Usuallly it is used in combination with MPV (misaligment of the pulmonary veins). You have defined the latter separately; this is ok, as one can explain, why this is a misnamed. We can define the former separately, because some Pathologists give the diagnosis of ACD without MPV.

So if we go with your definition above, I would modfiy to:

1
Definition (free text, please give PubMed ID)
A developmental defect of the alveolar airspace during cananicular lung development characterized by a decreased number of pulmonary capillaries located away from the alveolar epithelium, thickened alveolar septae, media hypertrophy of small pulmonary arteries and muscularization of distal arterioles, and in some cases lymphangiectasis. PMID: 21471096. 

Would not group under vascular abnormality; this is to group under:
Use parent term
Aplasia/Hypoplasia of the lungs HP:0006703
However we need a closer work up of these conditions, as improtant and associated with many genes.

2
I would also / separately define ACD/MPV 

ACD/MPV is characterized by absence or reduced pulmonary veins in the interlobular septa, with veins malpositioned adjacent to the arteries in the bronchovascular regions. The alveoli may be enlarged with thickened septa and few, but large, centrally placed capillaries without good approximation to the alveolus. Pulmonary arteries show marked hypertensive change with thickened muscular walls. PMID: 31189067.

Use parent term
Aplasia/Hypoplasia of the lungs HP:0006703

3
New term:
Acinar dysplasia

Synonyms: Acinar aplasia, Cystic pulmonary airway malformation type 0

Definition: 
Arrest of lung development in the pseudoglandular stage (weeks 8 to 16 of human gestation) resulting in small sized lungs, predominantly composed of bronchial and bronchiolar structures embedded in abundant loose mesenchyme that is
poorly vascularized. Acinar structures are essentially absent, with no significant formation of saccules and no alveoli. PMID: 31189067; PMID: 31151956

Gene associations: 
TBX4; PMID: 31151956, PMID: 30639323
FGF10; PMID: 30639323
FGFR2; PMID: 30639323

Use parent term
Aplasia/Hypoplasia of the lungs HP:0006703

4
New term:
Congenital alveolar dysplasia, CAD

Definition
Arrest of lung development in the cananicular stage (weeks 18 to 26 of human gestation) resulting in simplified acinar spaces, frequently with abundant intervening mesenchyme and no alveoli. In later arrest growth stages early saccular formations may be seen. May resemble the lobular maldevelopment often seen in ACDMPV without "vein misalignment" or marked hypertensive changes of the pulmonary arteries. PMID: 31189067.

Gene associations: 
TBX4; PMID: 31151956, PMID: 30639323
FGF10; PMID: 30639323

Parent term
Aplasia/Hypoplasia of the lungs HP:0006703

**Comment by pnrobinson** [comment]
added, thanks!
